docs/README.md

```diff
@@ -1,58 +1,78 @@
+This repository has the blocks that you should use to create link in the store
 
-We use `yarn` as our default package manager, before coding make sure to run yarn on: `root` and `react` folders.
+![image](https://user-images.githubusercontent.com/8517023/73387868-f1b36f80-42af-11ea-8e24-3045d2c819b4.png)
 
-## Some features:
+## Configuration
 
-### Tests
+1. Import the store link's app to your theme's dependencies in the `manifest.json`, for example:
 
-For testing we use `@vtex/test-tools`, our own testing framework based on `react-testing-library`, the tests should be located on the `react/__tests__` folder. For references, visit our [repository](https://github.com/vtex/test-tools).
-
-### Hooks
-
-Husky hooks tha runs on every `pre-commit` and `pre-push`.
-
-### Intl Equalizer
-
-Tool for equalizing the messages located on the `messages` folder/builder. It's configured to use the **en.json** as the default file for comparison. For references, visit our [repository](https://github.com/vtex/intl-equalizer).
-
-### Lint + Formatting
-
-TS lint configured with Prettier and .Config.
+```jsonc
+{
+  "vtex.store-link": "0.x"
+}
+```
 
-### Available Scripts
+2. Now you can use the block that the `store-link` app export. In the following example you can see the usage of the `store-link.product` inside of a quickview component:
 
-```json
+```jsonc
 {
-  "lint": "cd ./react && yarn lint",
-  "test": "cd ./react && yarn test",
-  "lint:locales": "intl-equalizer",
-  "locales:fix": "intl-equalizer --fix",
-  "verify": "yarn lint && yarn lint:locales && yarn test"
+  "store-link.product#quickview": {
+    "props": {
+      "href": "/{slug}/p",
+      "label": "Mais detalhes >",
+      "blockClass": "quickview"
+    }
+  },
+  "flex-layout.col#quickview-product-information": {
+    "children": [
+      "vtex.store-components:product-name",
+      "product-rating-summary",
+      "product-price#quickview-price",
+      "product-separator",
+      "product-identifier.product",
+      "store-link.product#quickview"
+    ],
+    "props": {
+      "preventVerticalStretch": true,
+      "rowGap": 0,
+      "width": "calc(50% - 0.375rem)"
+    }
+  },
+  "product-price#quickview-price": {
+    "props": {
+      "showInstallments": true,
+      "showSavings": true
+    }
+  }
 }
 ```
 
-### Ci
+You can see that it is passing the href prop with a `{slug}` thing, that will take the value of the product context that it is inserted and use it to generate a link like `/everyday-necessaire/p`. But you need to have in mind that in order to this work you have to put the `store-link.product` block inside of a another block that provides a product context. In this case the [ProductSummary](https://vtex.io/docs/components/product/vtex.product-summary) provides a product context and the `modal-layout` is included the `product-summary` and the `store-link.product` is included in the `modal-layout` that is why it has the product context.
```

Comment:
```suggestion
You can see that there is a `{slug}` placeholder being passed to the `href` prop. When rendered, the placeholder will be replaced with the respective value of the closest product context and generate a link like `/everyday-necessaire/p`. However, keep in mind that in order for this work, you have to place the `store-link.product` block inside of a another block that provides a product context, such as the [`ProductSummary`](https://vtex.io/docs/components/product/vtex.product-summary) or the `modal-layout`, which already includes a `product-summary` block.
```

Reply:
> 

"which already includes a `product-summary` block", the `modal-layout` is included in the `product-summary`